scipy/stats/tests/test_stats.py

```diff
@@ -501,6 +505,15 @@ class TestCorrSpearmanr(TestCase):
         other variables.  The same should go for SPEARMAN corelations, if
         your program has them.
     """
+    def test_scalar(self):
+        y = stats.spearmanr(4.)
+        assert_(np.isnan(y[0]))
+        assert_(np.isnan(y[1]))
```

Comment:
I've made the change here. I decided to keep the long version for the `ttest_*` ones for consistency with the rest of those test cases.